.circleci/config.yml

```diff
@@ -0,0 +1,39 @@
+version: 2.1
+commands:
+  update-npm:
+    steps:
+      - run: sudo npm install -g npm@latest
+  install-npm-wee:
+    steps:
+      - run: npm ci
+  build:
+    steps:
+      - run: npm run tsc
+  deploy-master-to-heroku:
+    steps:
+      - run: |
+          git push https://heroku:$HEROKU_API_KEY@git.heroku.com/$HEROKU_APP_NAME.git master
+jobs:
+  build:
+    working_directory: ~/EPIC-BULLETIN-BOARD-PROJECT
+    docker:
+      - image: circleci/node:12.5.0
+    steps:
+      - checkout
+      - update-npm
+      - install-npm-wee
+      - build
+  deploy:
+    docker:
+      - image: buildpack-deps:trusty
```

Comment:
Hmm you should use same images 🤔 Check from circleci docs about executors. There you can define same image you use always